ros_ws/src/crazyswarm/src/crazyswarm_server.cpp

```diff
@@ -608,40 +625,6 @@ class CrazyflieROS
     ROS_INFO("[%s] Ready. Elapsed: %f s", m_frame.c_str(), elapsedSeconds.count());
   }
 
-  void onLinkQuality(float linkQuality) {
-      if (linkQuality < 0.7) {
-        ROS_WARN("[%s] Link Quality low (%f)", m_frame.c_str(), linkQuality);
-      }
-  }
-
-  void onConsole(const char* msg) {
-    m_messageBuffer += msg;
-    size_t pos = m_messageBuffer.find('\n');
-    if (pos != std::string::npos) {
-      m_messageBuffer[pos] = 0;
-      ROS_INFO("[%s] %s", m_frame.c_str(), m_messageBuffer.c_str());
-      m_messageBuffer.erase(0, pos+1);
-    }
-  }
-
-
-  void onLogCustom(uint32_t time_in_ms, std::vector<double>* values, void* userData) {
-
-    ros::Publisher* pub = reinterpret_cast<ros::Publisher*>(userData);
-
-    crazyflie_driver::GenericLogData msg;
-    msg.header.stamp = ros::Time(time_in_ms/1000.0);
-    msg.values = *values;
-
-    m_logFile << time_in_ms / 1000.0 << ",";
-    for (const auto& value : *values) {
-      m_logFile << value << ",";
-    }
-    m_logFile << std::endl;
-
-    pub->publish(msg);
-  }
-
```

Comment:
are you moving these big blocks so they are more appropriately grouped with other methods/data?

Reply:
Yes, they are also now marked "private:"